packages/b/blosc/xmake.lua

```diff
@@ -0,0 +1,44 @@
+package("blosc")
+
+    set_homepage("https://www.blosc.org/")
+    set_description("A blocking, shuffling and loss-less compression library")
+
+    add_urls("https://github.com/Blosc/c-blosc/archive/v$(version).tar.gz",
+             "https://github.com/Blosc/c-blosc")
+
+    add_versions("1.20.1", "42c4d3fcce52af9f8e2078f8f57681bfc711706a3330cb72b9b39e05ae18a413")
+
+    add_deps("cmake")
+
+    on_install("macosx", "linux", "windows", "mingw", function (package)
+        configs = {}
+        table.insert(configs, "-DCMAKE_BUILD_TYPE=" .. (package:debug() and "Debug" or "Release"))
+        import("package.tools.cmake").install(package, configs)
+        if is_plat("windows") then
+            os.rm(path.join(package:installdir("lib"), "blosc.lib"))
+        elseif is_plat("mingw") then
+            os.rm(path.join(package:installdir("lib"), "libblosc.dll.a"))
+        end
```

Comment:
we need not remove these files. xmake will automatically clean up these cache files.

Reply:
These files provide "address to dynamic link library", and will be useful only if the `.dll` files is in the executable search directories or we use something like gcc -rpath flag during link stage, which is unfortunately disabled on Windows. Otherwise the executable can be compiled normally but not run. Tests on my local environment have shown, since compiler flag `-lblosc` prefer these files, I can't link to the true static libraries which are named 'libblosc.lib' and 'libblosc.a'. That's why I removed these files manually, and I wonder if there is a better solution.

Reply:
You should also deal with package:config("shared") and disable tests

https://github.com/Blosc/c-blosc/blob/5352508a420bec865c57cda116a5e5303df898d2/CMakeLists.txt#L93-L98

Reply:
```lua
local configs = {"-DBUILD_TESTS=OFF"}
table.insert(configs, "-DCMAKE_BUILD_TYPE=" .. (package:debug() and "Debug" or "Release"))
if package:config("shared") then
        table.insert(configs, "-DBUILD_SHARED=ON")
        table.insert(configs, "-DBUILD_STATIC=OFF")
else
        table.insert(configs, "-DBUILD_SHARED=OFF")
        table.insert(configs, "-DBUILD_STATIC=ON")
end
```

Reply:
It works. Thank you!